coding-concepts.js

```diff
@@ -15,45 +15,48 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 
 // --------------------1) What will this log?
 
-const cohort = "LEARN 2022"
+// const cohort = "LEARN 2022"
 // console.log(cohort.length)
 
-// a) Your answer:
+// a) Your answer: The length of "LEARN 2022", which is 10.
 // b) Verify and explain:
 
-
+// .length counts how many characters are in the value. It also will count spaces. I can also use it an Array its not limited to a string.
 // --------------------2) What will this log?
 
-const greeting = "Hello World!"
+// const greeting = "Hello World!"
 // console.log(greeting[4])
 
-// a) Your answer:
+// a) Your answer: It would log 'o'.
 // b) Verify and explain:
 
+// It logs the letter 'o' because you are using [4] to pick the value at the 4th index. And it is using the zero index. Which starts with 'H' being 0.
 
 // --------------------3) What will this log?
 
-const languages = ["JavaScript", "Ruby", "Python", "C++"]
-const index = 1
+// const languages = ["JavaScript", "Ruby", "Python", "C++"]
+// const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
+// a) Your answer: It would return "JavaScript".
 // b) Verify and explain:
-
+// It logged "Ruby" because each string is its own element in the Array. It would count 0,1,2,3 because of the zero-index. And the index [] is being used to insert the value of 1 from the declared variable index.
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+console.log(weekendDays[0,1].toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: It would log SATURDAY, SUNDAY in the terminal
+// b) Verify and explain: 
+// It gave an error and didn't log as I thought it would right away because I need to pick the index. I need to specify because .toUpperCase is a string method. But the values are strings inside of an Array so I need to tell it which index when using it an a Array. .toUpperCase also capitalizes every character when used.
```

Comment:
stating the error was sufficient. appreciate that you did the work to show how to properly make it all caps